test/integration/family-suite/90 - Declare YM2 resources scenario.js

```diff
@@ -3,7 +3,7 @@ description: 'Declare YM2 resources',
 steps: [
     ResultComponent.declareYM2Resources(),
     {
-        'YM2ResourcesComponent.suggestionText': /12000 €/,
+        'YM2ResourcesComponent.suggestionText': /12,000.00 €/,
```

Comment:
This is not the proper notation for currencies in France. The thousands separator should be ` `, and the comma should be `,`.
If you want to use the native `number` filter (which is probably a good idea — unless it should be [`currency`](https://docs.angularjs.org/api/ng/filter/currency) so we don't have to repeat `€`? 😉), then you will need to include [Angular's `i18n`](https://docs.angularjs.org/guide/i18n).